react/components/EXPERIMENTAL_Table/README.md

```diff
@@ -642,6 +642,1091 @@ function VisibilityExample() {
 ;<VisibilityExample />
 ```
 
+# Features
+
+<div className="center mw7 pv6">
+ ![](./table.png)
+</div>
+
+## Toolbar
+
+The toolbar is a bundle of features, including search input, autocomplete, columns visibility toggle, density controls, import and export buttons, extra actions menu using ActionMenu component and a newLine button to help with entry creation (you can see the illustrative diagram in the beginning of the page for a better visualization of this structure)
+
+#### InputSearch
+
+- A wrapper around `InputSearch` component. The props are the same.
+
+#### InputAutocomplete
+
+- A wrapper around `AutocompleteInput` component. The props are the same.
+
+#### Button Group
+
+Represents the group of buttons located at the right. It has other composites that are described down below.
+
+##### Columns
+
+- A button that toggles columns visibility.
+- It is recommended to combine it with the `useTableVisiblitity` hook.
```

Comment:
```suggestion
- It should be used with the `useTableVisibility` hook.
```
Would be great if styleguidist allowed to link sections of the same file :/

Reply:
You can achieve visibility without using the hook, actually :)